### ISSUE TITLE
Fix the twilight forest incompability 

### ISSUE THREAD

**Issue by CandyPillager** [sp614x/optifine#5825]
## Description of Issue

As you probably already know, Twilight Forest and Optifine are incompatible in 1.16.. the Twilight Forest's mobs don't render correctly when using OF. this is really annoying since Twilight Forest is my favorite mod and I've been waiting for it's port for over a year, just to discover it doesn't work with OF. I kinda need to use OF since my laptop isn't amazing, and especially in mod packs I need OF for a performance boost. 

## Steps to Reproduce
1. Install optifine, install Twilight forest 1.16.5, load the game and spawn in some of the bosses. (Hydra, Snow Queen, Naga). Their models are glitched. the Hydra's 3 heads don't render, the Naga's tail doesn't render and the Snow Queen's ice cage doesn't render. I haven't tested many more things, but Twilight's devs say it's not their fault. 
2. ...to find this bug?

**Comment by CandyPillager** [comment]
Optifine Version: 1.16.5 G5
Installation method: Forge put in mods folder
Other installed mods: Twilight Forest
Prior: doesn't happen without optifine and with only twilight forest.

Apparently I placed my answers wrong in the boxes... so this is the info

**Comment by KaiAF** [comment]
Are you able to send a F3 debug menu screenshot? This might be able to help

**Comment by CandyPillager** [comment]
> Are you able to send a F3 debug menu screenshot? This might be able to help

Here you go. Sorry for the late response 

![Screenshot (669)](https://user-images.githubusercontent.com/62050983/122806827-1bb56980-d2cb-11eb-959e-c24e439f5227.png)
![Screenshot (670)](https://user-images.githubusercontent.com/62050983/122806831-1ce69680-d2cb-11eb-9443-4d5ef5a5e830.png)

**Comment by officialjrem** [comment]
Any updates on this issue?

**Comment by Breadabix** [comment]
I dont know about the bosses but the latest 1.16 versions of the mods combined still gives broken rainbow oak leaves, they just look normal. So there are still compatibility issues.

**Comment by LuNeder** [comment]
Still seems to be a problem on latest versions, according to Twilight’s Discord

**Comment by SeaOfPixels** [comment]
Confirmed for OptiFine 1.20.4 HD U I8 pre1, according to the Twilight Forest team OptiFine removes a vanilla feature that the Twilight Forest needs, causing crashes and broken behavior

**Comment by Chuzume** [comment]
I don't think you need to stick with OptiFine, but why would you want to use it? There's an official statement that Twilight Forest is not compatible with OptiFine.
